packages/bpk-component-nudger/src/BpkNudger.js

```diff
@@ -16,110 +16,53 @@
  * limitations under the License.
  */
 
-import PropTypes from 'prop-types';
+/* @flow strict */
+
 import React from 'react';
-import BpkButton from 'bpk-component-button';
-import { withButtonAlignment } from 'bpk-component-icon';
-import MinusIcon from 'bpk-component-icon/sm/minus';
-import PlusIcon from 'bpk-component-icon/sm/plus';
+import PropTypes from 'prop-types';
 import { cssModules } from 'bpk-react-utils';
-import clamp from 'lodash.clamp';
 
+import BpkConfigurableNudger from './BpkConfigurableNudger';
+import {
+  type CommonProps,
+  COMMON_DEFAULT_PROPS,
+  COMMON_PROP_TYPES,
+} from './common-types';
 import STYLES from './BpkNudger.scss';
 
-const getClassName = cssModules(STYLES);
-
-const AlignedMinusIcon = withButtonAlignment(MinusIcon);
-const AlignedPlusIcon = withButtonAlignment(PlusIcon);
-
-const BpkNudger = props => {
-  const {
-    id,
-    min,
-    max,
-    value,
-    onChange,
-    className,
-    increaseButtonLabel,
-    decreaseButtonLabel,
-    buttonType,
-  } = props;
-  const classNames = [getClassName('bpk-nudger')];
-  if (className) {
-    classNames.push(className);
-  }
+type Props = {
+  ...$Exact<CommonProps<number>>,
+};
 
-  const adjustedValue = Math.floor(clamp(value, min, max));
-  const decreaseDisabled = adjustedValue <= min;
-  const increaseDisabled = adjustedValue >= max;
+const getClassName = cssModules(STYLES);
 
-  const minusIconClassNames = [getClassName('bpk-nudger__icon')];
-  if (decreaseDisabled) {
-    minusIconClassNames.push(getClassName('bpk-nudger__icon--disabled'));
-  }
-  const plusIconClassNames = [getClassName('bpk-nudger__icon')];
-  if (increaseDisabled) {
-    plusIconClassNames.push(getClassName('bpk-nudger__icon--disabled'));
-  }
+const BpkNudger = (props: Props) => {
+  const { ...rest } = props;
 
-  const inputStyles = [getClassName('bpk-nudger__input')];
-  if (buttonType === 'outline') {
-    inputStyles.push(getClassName('bpk-nudger__input--outline'));
-  }
+  const compareValues = (a: number, b: number): number => a - b;
+  const incrementValue = (currentValue: number): number => currentValue + 1;
```

Comment:
These don't need to be declared inside the `render` function. Doing so means they are needlessly redelcared on each render and might also confuse the reader by making it look like they are closures over some value which they are not